<launch>
<node pkg="imu_complementary_filter" type="complementary_filter_node"
      name="complementary_filter_gain_node" output="screen">
    <remap from="IMU" to="imu/data_raw"/>
    <param name="fixed_frame" value="imu"/>
    <param name="publish_tf" value="true"/>
    <param name="do_bias_estimation" value="true"/>
    <param name="bias_alpha" value="0.05"/>
    <param name="do_adaptive_gain" value="false"/>
    <param name="use_mag" value="false"/>
    <param name="gain_acc" value="0.0007"/>
    <param name="gain_mag" value="0.01"/>
</node>
</launch>
